include/zephyr/bluetooth/conn.h

```diff
@@ -1052,6 +1131,22 @@ struct bt_conn_cb {
 			      const struct bt_df_conn_iq_samples_report *iq_report);
 #endif /* CONFIG_BT_DF_CONNECTION_CTE_RX */
 
+#if defined(CONFIG_BT_TRANSMIT_POWER_CONTROL)
+	/** @brief LE Read Remote Transmit Power Level procedure has completed or LE
+	 *  Transmit Power Reporting event.
+	 *
+	 *  This callback notifies the application that either the remote transmit power level
+	 *  has been read from the peer or transmit power level has changed for the local or
+	 *  remote controller when transmit power reporting is enabled for the respective side
+	 *  using @ref bt_conn_le_set_tx_power_report_enable.
+	 *
+	 *  @param conn Connection object.
+	 *  @param report Transmit power report.
+	 */
+	void (*tx_power_report)(struct bt_conn *conn,
+				      const struct bt_conn_le_tx_power_report *report);
```

Comment:
minor: the split line doesn't seem to be aligned consistently (to the other similar places in this file, or Bluetooth subsystem code for that matter). Should be n tabs + 0-7 spaces to align right after the `(` of the previous line.

Reply:
FWIW, this seems to be the only place with this issue - the alignment looks correct in all other "split line" places in this PR

Reply:
Will change soon. Thanks :)